src/combobox/combobox.js

```diff
@@ -170,7 +173,7 @@ function Combobox<OptionT>(props: PropsT<OptionT>) {
       const stringified = mapOptionToString(clickedOption);
       setIsOpen(false);
       setSelectionIndex(index);
-      onChange(stringified);
+      onChange(stringified, clickedOption);
```

Comment:
Would you mind adding a comment for v10 to change the `onChange` handler to be aligned with the rest of the components and use a type like
```
onChange: (params: {
  value: ValueT,
  type: ChangeSourceT,
}) => mixed,
```

Reply:
👍 added a comment in the `types.js` file